Cloud Docker release notes 1.2.1 (#8404)
magento/devdocs: src/cloud/release-notes/mcd-release-notes.md

-  {:.new}New features
-  {:.fix}Fixes and improvements

## v1.2.1
*Release date: December 21, 2020*<br/>

-  {:.fix}Fixed PHP-FPM container health check by changing the `WEB_PORT` from `80` to `8080`.<!--MCLOUD-7232-->

-  {:.fix}Fixed an error with invalid volume naming in developer mode.<!--MCLOUD-7442-->

> -  {:.new}Added the ability to change the value of the parameters "worker_processes" (option "--nginx-worker-processes" ) and "worker_connections" (option "--nginx-worker-processes" ) for services "tls" and "web". The parameter 'worker_process' retains the ability to set the value "auto".

Reviewer: What do these options do?  That is, why do you need to add them.  Can you provide link to more information?  Are they options for the `ece-docker build:compos` command?

```suggestion
-  {:.new}Added the ability to change the value of the parameters `worker_processes` (option `--nginx-worker-processes`) and `worker_connections` (option `--nginx-worker-processes` ) for the tls and web services . The parameter 'worker_process' retains the ability to set the value `auto`.
```

> -  {:.new}Added options to change the value of NGINX `worker_processes` (option: `--nginx-worker-processes`) and NGINX `worker_connections` (option: `--nginx-worker-processes` ) for tls and web services. The parameter `worker_process` retains the ability to set the value `auto`.<!--MCLOUD-7259-->

Reviewer 2: Should probably indicate these are options for the `build` command. Same for next line, too.

> -  {:.new}Added the ability to create a configuration without the "tls" service.

Reviewer: ```suggestion
-  {:.new}Added the ability to create a configuration without the tls service.
```

> -  {:.new}Reduced the value of memory consumed by the NGINX process for 'tls' and 'web' services

Reviewer: ```suggestion
-  {:.new}Reduced the value of memory consumed by the NGINX process for tls and web services
```

-  {:.new}New features
-  {:.fix}Fixes and improvements

## v1.2.1
*Release date: December 21, 2020*<br/>

-  {:.new}<!--MCLOUD-7259-->**NGINX command options.** Added build command options to change the number of NGINX `worker_processes` and NGINX `worker_connections` for TLS and Web services. The `worker_process` parameter retains the ability to set the value to `auto`. Examples:

    ```bash

>     ./vendor/bin/ece-docker build:compose --nginx-worker-processes=2

Reviewer 2: So, placing two commands in a bash code block makes the copy command ineffective for copying a single line. Either use two separate code blocks, or use a different determiner than bash so that it's more like a list of commands than a command to copy and run.
